internal/validate/image_test.go

```diff
@@ -74,21 +77,24 @@ func Test_Validate_ImageWhichIsNotInNotary_ShouldReturnError(t *testing.T) {
 	s := NewDefaultMockNotaryService().WithFunc(f).Build()
 	err := s.Validate(context.TODO(), UntrustedImageName)
 	require.Error(t, err)
-	require.ErrorContains(t, err, "does not have trust data for")
+	require.Contains(t, err.Error(), "does not have trust data for")
```

Comment:
We can replcae `require.Error` and `require.Contains` with `require.ErrorContaints`